proto/spire/api/server/agent/v1/agent.proto

```diff
@@ -77,6 +77,9 @@ message ListAgentsRequest {
 
         // Filters agents to those that are banned.
         google.protobuf.BoolValue by_banned = 3;
+
+        // Filters agents that can re-attest.
+        google.protobuf.BoolValue by_can_re_attest = 4;
```

Comment:
nit: Would eliminate the `_` between `re` and `attest` 

```suggestion
        google.protobuf.BoolValue by_can_reattest = 4;
```

Reply:
Thank you for the review, @faisal-memon! I've addressed this suggestion in 
990fa1d 😄